tests/composition_names_duplicates.prm

```diff
@@ -0,0 +1,109 @@
+# A test to ensure the naming of compositional fields works as expected.
+
+# EXPECT FAILURE
```

Comment:
You should make motivational posters ;-)

Reply:
Assume the worst then you will always be positively surprised :-)
But in this case it is a macro that tells the tester to pass if the test fails, and fail if the test passes.

Reply:
I take credit for this brilliant feature including the string. :-)